prime-dotnet-webapi/Configuration/Database/Seeded Tables/RemoteAccessTypeLicenseConfiguration.cs

```diff
@@ -0,0 +1,84 @@
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Prime.Models;
+
+namespace Prime.Configuration.Database
+{
+    public class RemoteAccessTypeLicenseConfiguration : SeededTable<RemoteAccessTypeLicense>
+    {
+        public override IEnumerable<RemoteAccessTypeLicense> SeedData
+        {
+            get
+            {
+                return new[] {
+                    // Normal Remote Access
```

Comment:
Revise to "Normal Remote Access for select physician and nurse practitioner license classes"

Reply:
Comment added.